ldclient/__init__.py

```diff
@@ -135,27 +139,19 @@ def __init__(self, api_key, config):
         self.daemon = True
         self._api_key = api_key
         self._config = config
-        self._store = InMemoryFeatureStore()
+        self._store = config._feature_store_class()
+        self._running = False
 
     def run(self):
         log.debug("Starting stream processor")
+        self._running = True
         hdrs = _stream_headers(self._api_key)
         uri = self._config._stream_uri + "/"    
```

Comment:
We're making one minor change to the streaming SDKs-- rather than subscribe to the root resource `https://stream.launchdarkly.com/`, we're going to subscribe to `https:/stream.launchdarkly.com/features`.

We're doing this because we're going to start adding other things to `stream.ld.com` that are not relevant to the SDKs, and the root resource is intended to subscribe you to everything. 

Besides changing the URI here, you'll also need to change the event names (e.g. `put/features` becomes `put`). Here's the change we made to ruby, as a reference:

https://github.com/launchdarkly/ruby-client/commit/e2aeae5ac21d6bf9a468b271515d3497dda2030d

I was going to do this myself, but since your PR is still out I'd either have to fork your fork, or make the change on master and make you deal with the conflict. I figure just having you do it is easiest.